board/drivers/can_common.h

```diff
@@ -160,7 +160,11 @@ void can_clear(can_ring *q) {
 uint8_t bus_lookup[] = {0,1,2};
 uint8_t can_num_lookup[] = {0,1,2,-1};
 uint32_t can_speed[] = {5000, 5000, 5000, 333};
-uint32_t can_data_speed[] = {5000, 5000, 5000}; //For CAN FD with BRS only
+// For CAN FD only
+uint32_t can_data_speed[] = {5000, 5000, 5000};
+bool canfd_enabled[] = {false, false, false};
+bool brs_enabled[] = {false, false, false};
```

Comment:
maybe we can refactor this a bit to a struct with the bus config and just have an array of that?